compiler/q-implant/README.md

```diff
@@ -0,0 +1,25 @@
+# q-implant
+
+_q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
+
+## Example
+
+```sh
+q-implant input.circle qparam.json output.circle
+```
+
+`qparam.json` and `*.npy` files must exist in the same directory. For example,
+
+```json
+{
+  "ifm": {
+    "dtype": "uint8",
+    "scale": "0.npy",
+    "zerop": "1.npy",
+    "quantized_dimension": 0
+  },
+  ...
+}
+```
+
+`0.npy` and `1.npy` must exist in the same directory with `qparam.json`.
```

Comment:
moved from https://github.com/Samsung/ONE/pull/11579#pullrequestreview-1656684905

from this explanation, there will be two `.npy` files for one operator(node) from the model. for real models, there could be 10s or 100s for nodes in a model.
which would produces lots of files in the process of `q-implant`.
I don't think this is a good approach for users.

How about using `h5` format and produce single file for a model?

Reply:
There is an issue in using h5 format.

tensors can have / in their names, but h5 does not allow / in the object name. So we have to replace it into another character, which makes hard to recover the original tensor name.

---
Copied from https://github.com/Samsung/ONE/pull/11579#discussion_r1345284188

Reply:
FYI, I used | character to use in replacement in our internal CI testing. not perfect but currently it's working.

Reply:
marked above comment as `off-topic` as there was some misunderstanding (README showed as an example)